templates/client-daemonset.yaml

```
@@ -219,8 +219,9 @@ spec:
                 {{- end }}
                 {{- else }}
                 {{- if .Values.server.enabled }}
+                {{- $serverSerflanPort  := .Values.server.serflan.port -}}
```

nit:
```suggestion
                {{- $serverSerfLANPort  := .Values.server.serflan.port -}}
```

golang style would be to uppercase an ancronym like this in the middle of the string I think